### PR TITLE
Update Kicad download; Update SumatraPDF download

### PR DESCRIPTION
**Pull request by Joly0**

### CODE REVIEW

**Comment by Joly0** [comment]
Btw i tried to fix filezilla-server, but it seems like its blocking the downloads, it should have the correct link, but it only gives back "unexpected response status: 403". Maybe its a good idea to print the link when using -l even when errors occur, which it seems like it doesnt.

**Review by mimi89999 — CHANGES_REQUESTED** [review]
Please rebase on master instead of merging master commits to your branch

**Comment by Joly0** [comment]
> Please rebase on master instead of merging master commits to your branch

Done

File: jiup/rules/rules.go [hunk 1]
```diff
@@ -833,12 +833,11 @@ func init() {
 	Rule("kicad",
 		v.Regexp(
 			"http://kicad-pcb.org/download/windows/",
```
Comment (mimi89999) [inline jiup/rules/rules.go:835]: Shouldn't it be changed to https://www.kicad.org/download/windows/ ?

File: jiup/rules/rules.go [hunk 2]
```diff
@@ -833,12 +833,11 @@ func init() {
...
-			h.Re("Current Version: <strong>([0-9.]+)</strong>"),
```
Comment (mimi89999) [inline jiup/rules/rules.go:836]: Why was this changed? On the website I see:
```
<p>Current Version: <strong>5.1.10</strong></p>
```

File: jiup/rules/rules.go (hunk 1 above)
Comment (Joly0) [reply jiup/rules/rules.go:835]: Thats right, idk where the "-pcb" came from, but it works, so that might be the answer, why i didnt see this error.

File: jiup/rules/rules.go (hunk 2 above)
Comment (Joly0) [reply jiup/rules/rules.go:836]: Because Kicad seems to add _[0-9] to the exe files after the version number which the old regexp didnt handle

File: jiup/rules/rules.go [hunk 3]
```diff
@@ -832,13 +832,12 @@ func init() {
...
+			"http://kicad.org/download/windows/",
+			h.Re("kicad-([0-9.]+_[0-9])-"),
 		),
-		d.HTMLA(
-			"http://kicad-pcb.org/download/windows/",
-			"a[href$='-i686.exe']",
-			"a[href$='-x86_64.exe']",
+		d.Template(
+			"https://osdn.net/frs/redir.php?m=dotsrc&f=/storage/g/k/ki/kicad/kicad-{{.Version}}-i686.exe",
+			"https://osdn.net/frs/redir.php?m=dotsrc&f=/storage/g/k/ki/kicad/kicad-{{.Version}}-i686.exe",
```
Comment (mimi89999) [inline jiup/rules/rules.go:840]: That's the 64 bit version

Comment (Joly0) [reply jiup/rules/rules.go]: Corrected it

**Comment by mimi89999** [comment]
Please squash the commits and change `Update/Fix` to either.

**Comment by Joly0** [comment]
> Please squash the commits and change `Update/Fix` to either.

K, done

**Review by mimi89999 — APPROVED** [review]

**Comment by mimi89999** [comment]
I proposed another solution in https://github.com/just-install/just-install-updater-go/pull/112
